executor/write.go

```diff
@@ -197,38 +197,40 @@ func (e *DeleteExec) Next(goCtx goctx.Context) (Row, error) {
 	return nil, e.deleteSingleTable(goCtx)
 }
 
+// NextChunk implements the Executor NextChunk interface.
+func (e *DeleteExec) NextChunk(goCtx goctx.Context, chk *chunk.Chunk) error {
+	chk.Reset()
+	if e.finished {
+		return nil
+	}
+	defer func() {
+		e.finished = true
+	}()
+
+	if e.IsMultiTable {
+		return e.deleteMultiTablesByChunk(goCtx)
+	}
+	return e.deleteSingleTableByChunk(goCtx)
+}
+
 type tblColPosInfo struct {
 	tblID         int64
 	colBeginIndex int
 	colEndIndex   int
 	handleIndex   int
 }
 
+// tableRowMapType is a map for unique (Table, Row) pair.
+type tableRowMapType map[int64]map[int64][]types.Datum
```

Comment:
add a comment to explain what the key(`int64`) and value(`map[int64][]types.Datum`) represent for.